lib/src/Uncertainty/Distribution/GeneralizedExtremeValue.cxx

```diff
@@ -333,6 +333,7 @@ Graph GeneralizedExtremeValue::drawReturnLevel() const
   drawable.setLegend("GEV return level");
   drawable.setLineStyle("solid");
   drawable.setLineWidth(2);
+  drawable.setColor("red");
```

Comment:
Is this necessary? Is the default color not satisfactory?

Reply:
yes the colors were mixed up with the new palette